### ISSUE TITLE
I need examples with code~

### ISSUE THREAD

**Issue by fishenal** [atlassian/react-beautiful-dnd#1916]
so hard for me, where to see the coe of the demos? storybook

**Comment by shaunstoltz** [comment]
Agreed, even the simplest attempt to use this library is exploding with hundreds of errors in console for a single list with 7 unique elements modified from example code...... and the massive complexity of even the most basic implementation renders this lib almost unusable...right?

**Comment by fishenal** [comment]
> Agreed, even the simplest attempt to use this library is exploding with hundreds of errors in console for a single list with 7 unique elements modified from example code...... and the massive complexity of even the most basic implementation renders this lib almost unusable...right?

yes, it's wired that so many examples(in storybook) without codes, it's very good at examples amount, but very bad at show codes. I don't think other lib like this, or I did'nt go to the right page?

**Comment by imjoshellis** [comment]
I agree, the storybook method used here is nice for presentation but not good for learning. The story code is in https://github.com/atlassian/react-beautiful-dnd/tree/master/stories/src, but you'll have to jump around a lot between modules to understand it all. 

I found it easier to learn from the walkthrough with videos (code in descriptions) on egghead: https://egghead.io/courses/beautiful-and-accessible-drag-and-drop-with-react-beautiful-dnd

Finally, this youtuber did a pretty good video walkthrough series as well (he has github code links in video description): https://www.youtube.com/playlist?list=PLN3n1USn4xlnsj993By8YSvsEl1VWhV8R

**Comment by kishorenaib** [comment]
Yeah,  why not list out an example and spec the API in the readme.md ?   Would be better than videos.  Need to take a glance at the API before investing time.
